src/app/rpmostree-builtin-status.c

```diff
@@ -563,6 +564,13 @@ status_generic (RPMOSTreeSysroot *sysroot_proxy,
           print_kv ("Unlocked", max_key_len, unlocked);
           g_print ("%s%s", bold_suffix, red_suffix);
         }
+      /* look for end of life attribute in the deployment */
+      g_variant_dict_lookup (dict, "end-of-life", "&s", &end_of_life_string);
+
+      if (end_of_life_string)
+        {
+          g_print("%s", end_of_life_string);
```

Comment:
Space between identifier and paren; we also usually don't do curly braces for single lines, but I'm fine keeping it too.